<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QWidget" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>PlayerA Level</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxA"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnAUp">
       <property name="text">
        <string>PlayerA LevelUp</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnMInfo">
       <property name="text">
        <string>Class MetaObject INFO</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>PlayerB Level</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxB"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnBUp">
       <property name="text">
        <string>PlayerB LevelUp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
